src/task_processor/routers.py

```diff
@@ -0,0 +1,77 @@
+from django.conf import settings
+from django.db.models import Model
+
+
+class TaskProcessorRouter:
+    """
+    Routing of database operations for task processor models
+    """
+
+    route_app_labels = ["task_processor"]
+
+    @property
+    def is_enabled(self) -> bool:
+        return "task_processor" in settings.TASK_PROCESSOR_DATABASES
+
+    def db_for_read(self, model: type[Model], **hints: None) -> str | None:
+        """
+        If enabled, route "task_processor" models to the a se database
```

Comment:
This doc string also looks wrong

Reply:
It was, and I can't grasp how, thanks for spotting it. Fixed in bda53cdac03bfea7edb045f3bdcfd13579e01bd2.